python/pecos/simulators/custatevec/bindings.py

```diff
@@ -51,6 +54,7 @@
     "SYY": two_q.SYY,
     "SYYdg": two_q.SYYdg,
     "SZZ": two_q.SZZ,
+    "SZZ": two_q.SZZ,

```

Comment:
```suggestion
```

Reply:
Fixed!  That Actually was supposed to have the SqrtZZ op.  Copy-paste error.